consensus/src/common.rs

```diff
@@ -176,6 +176,21 @@ impl Consensus for ProtocolStateChain {
                 // short-range fork, select longer chain
                 self.select_longer_chain(candidate)
             } else {
+                // check against sub window density sizes > 11
+                let tip_state = self
+                    .consensus_state()
+                    .ok_or(ConsensusError::ConsensusStateNotFound)?;
+                let candidate_state = candidate
+                    .consensus_state()
+                    .ok_or(ConsensusError::ConsensusStateNotFound)?;
+
+                let sub_windows_per_window = self.config().sub_windows_per_window.0 as usize;
+                if tip_state.sub_window_densities.len() != sub_windows_per_window {
+                    return Ok(candidate);
+                } else if candidate_state.sub_window_densities.len() != sub_windows_per_window {
+                    return Ok(self);
+                }
+
```

Comment:
@creativcoder since this is a part which impacts `relative_min_window_density`, adding this check at the starting of `relative_min_window_density` could be a better idea 
also as there are additional cases which we need to handle in this method, like extremely high sub window density
It makes more sense to me in adding it inside `relative_min_window_density`

Reply:
I had it there within relative_min_window_density before, but relative_min_window_density's returns a u32, whereas we need to return the chain when the lengths are not ideal. As a result, had to move it above.